### ISSUE TITLE
Incorporate issue creation mode

### ISSUE THREAD

**Issue by simar7** [aquasecurity/trivy-action#110]
@knqyf263 has created https://github.com/knqyf263/trivy-issue-action which can create issues by running trivy with GH Action. As we already have Trivy Action, we can incorporate that functionality here.

**Comment by b34rd-tek** [comment]
I find this action works better.

https://github.com/CemitDigital/trivy-report-issue-action

The downside is that if there is no 'Results' key found in the json file it will fail. I created an issue and a pull request on their repo but haven't seen any activity. I may just fork the repository and edit this myself :-).

**Comment by simar7** [comment]
hi @b34rd-tek - is there something missing in https://github.com/knqyf263/trivy-issue-action that we can add?

**Comment by b34rd-tek** [comment]
When I attempted to use the action you linked it wouldn't authenticate to gh to create the issues.

**Comment by saerosV** [comment]
@simar7 @b34rd-tek

I think the features added by [this PR](https://github.com/aquasecurity/trivy/pull/3202) ([related issue](https://github.com/aquasecurity/trivy/issues/3201)) could partially replace the [trivy-report-issue-action](https://github.com/CemitDigital/trivy-report-issue-action) tool, with the added benefit of not needing to use an external Action/tool.

**Comment by b34rd-tek** [comment]
I ended up re-writing the way Trivy was used after updating the action attached. The action from Aquasec is what I use now and I submit the reports to DefectDojo for tracking. I also have the action configured to generate SBOMs.

**Comment by saerosV** [comment]
Thanks for the follow-up, @b34rd-tek. DefectDojo seems to be one of the best ways of visualizing the reports.

> The action from Aquasec

Just to confirm, the action mentioned is Trivy Action, right?

**Comment by b34rd-tek** [comment]
Correct. I use that action to generate everything and add a step to post the results to defectdodjo.

**Comment by saerosV** [comment]
Got it, thanks again!
